pkgs/by-name/sp/spooftooph/package.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, stdenv
+, fetchurl
+, bluez
+, bluez-tools
+, ncurses
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "spooftooph";
+  version = "0.5.2";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/project/spooftooph/spooftooph-${finalAttrs.version}/spooftooph-${finalAttrs.version}.tar.gz";
+    hash = "sha256-JH5+fHpe83NJV9AR5MXKnrwaTqz4s2BGAcczbddVNHw=";
+  };
+
+  buildInputs = [
+    bluez
+    bluez-tools
+    ncurses
+  ];
+
+  postPatch = ''
+    substituteInPlace makefile \
+      --replace /usr/bin $out/bin
```

Comment:
```suggestion
      --replace-fail /usr/bin $out/bin
```

Reply:
What is the difference between `--replace` and `--replace-fail`?

Reply:
See #260776

Reply:
Thank you very much ^^